WeakAurasOptions/Cache.lua

```diff
@@ -277,19 +277,28 @@ function spellCache.BestKeyMatch(nearkey)
   return bestKey;
 end
 
+---@param input string | number
+---@return string name, number? id
 function spellCache.CorrectAuraName(input)
   if (not cache) then
     error("spellCache has not been loaded. Call WeakAuras.spellCache.Load(...) first.")
   end
 
-  local spellId = WeakAuras.SafeToNumber(input);
+  local spellId = WeakAuras.SafeToNumber(input)
+  if type(input) == "string" and input:find("|", nil, true) then
+    spellId = WeakAuras.SafeToNumber(input:match("|Hspell:(%d+)"))
+    print(input:gsub("|", "||"), spellId)
```

Comment:
there's a print here

Reply:
whups

Reply:
Well... You removed the other prints but not this one.-

Reply:
im practicing writing code for details addon